redaxo/src/core/lib/autoload.php

```diff
@@ -102,7 +102,10 @@ public static function autoload($class)
 
         // Class not found, so reanalyse all directories if not already done or if $force==true
         // but only if an admin is logged in
-        if ((!self::$reloaded || $force) && ($user = rex_backend_login::createUser()) && $user->isAdmin()) {
+        if (
+            (!self::$reloaded || $force) &&
+            (rex::isDebugMode() || ($user = rex_backend_login::createUser()) && $user->isAdmin())
```

Comment:
ich denke da man um den debug mode zu aktiveren sowieso admin sein muss, weiß ich nicht in welchem fall diese neue bedingung überhaupt helfen könnte